timvt/endpoints/factory.py

```diff
@@ -128,8 +128,8 @@ async def tilejson(
                 "y": "{y}",
             }
             tile_endpoint = self.url_for(request, "tile", **kwargs).replace("\\", "")
-            minzoom = minzoom or tms.minzoom
-            maxzoom = maxzoom or tms.maxzoom
+            minzoom = minzoom if minzoom is not None else (table.minzoom or tms.minzoom)
+            maxzoom = maxzoom if maxzoom is not None else (table.maxzoom or tms.maxzoom)
```

Comment:
there is edge cases where `table.minzoom` is `0` so it will default to `tms.minzoom` 🤷‍♂️ (max zoom should always be >0)